teams/teams-ps/teams/Set-CsTeamsShiftsConnectionInstance.md

```diff
@@ -241,97 +158,174 @@ Gets or sets the list of connector admin email addresses.
 Type: String[]
 Parameter Sets: SetExpanded, SetViaIdentityExpanded
 Aliases:
-
 Required: False
 Position: Named
 Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -ConnectorId
-The connector id
+### -ConnectorInstanceId
+The Id of the connector instance to be updated.
 
 ```yaml
 Type: String
-Parameter Sets: SetExpanded, SetViaIdentityExpanded
+Parameter Sets: SetExpanded, Set
 Aliases:
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
 
+### -ConnectionId
+Gets or sets the WFM connection id for the new instance. This can be retrieved by running [Get-CsTeamsShiftsConnection](Get-CsTeamsShiftsConnection.md).
```

Comment:
```suggestion
Gets or sets the WFM connection ID for the new instance. This can be retrieved by running [Get-CsTeamsShiftsConnection](Get-CsTeamsShiftsConnection.md).
```